src/endpoints/createFeedbackEndpoint.js

```diff
@@ -16,13 +16,19 @@ export const CATEGORIES_FEEDBACK_TYPE = 'categories'
 export const EVENTS_FEEDBACK_TYPE = 'events'
 export const EXTRAS_FEEDBACK_TYPE = 'extras'
 
+export const CONTENT_FEEDBACK_CATEGORY = 'Inhalte'
+export const TECHNICAL_FEEDBACK_CATEGORY = 'Technisches Feedback'
+
 export type FeedbackType = null | 'extra' | 'search' | 'categories' | 'events' | 'extras'
 
+export type FeedbackCategoryType = null | 'Inhalte' | 'Technisches Feedback'
```

Comment:
Why is this allowed to be null?

Reply:
Also I think the keys should be english and lower case as the others (`contents` instead of `Inhalte` and `technical` instead of `Technisches Feedback`)

Reply:
The keys are german because the cms needs them to be in german: "We do not yet support translated categories in the backend, so to ensure a consistent user experience, I propose to send the German names of the categories."

Reply:
@Simple-codinger what is the source of this quote?

Reply:
In the wiki of the api the german keys are used, so that is probably fine (https://github.com/Integreat/cms/wiki/REST-APIv3-Documentation#feedback). However, I still don't get why it is allowed to be null :see_no_evil: